spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/BlockingQueueConsumer.java

```diff
@@ -86,7 +84,7 @@
  * @author Alex Panchenko
  * @author Johno Crawford
  */
-public class BlockingQueueConsumer implements RecoveryListener {
+public class BlockingQueueConsumer {
```

Comment:
This is some kind of breaking change for back-porting...

Reply:
Why do you consider this a breaking change? The listener implementation was just moved to another class.

Reply:
Well, I mean an interaction exactly with this class, if someone somehow extract it from the container and cast to the `RecoveryListener` for his/her own peculiar logic.

But yeah... Should not be so critical do not proceed with the fix you have done so far...